client/routing.js

```diff
@@ -10,6 +10,11 @@ const STATE = {
   DONE: 4,
 }
 
+// Based on experimentation, we've found that the bounty
+// needs to be at least `profitabilityIndex * currentGasPrice`
+// to be profitable.
+const PROFITABILITY_INDEX = 100000;
```

Comment:
I don't think I understand the difference between the PROFITABILITY_INDEX and roughly the gas it costs to send the claim transaction. I think it's redundant.

Reply:
I think here we're trying to stop TimeNodes from claiming transactions that will not bring enough bounty to be profitable after execution. @kosecki123 can probably explain it a bit better.